lib/check-node.js

```diff
@@ -3,10 +3,10 @@ const semver = require('semver')
 
 module.exports = (cb) => {
   exec('node --version', (error, stdout, stderr) => {
-    if (error) return cb(error)
+    if (error) return cb(error, null)
     const output = (stdout + stderr).replace(/(\r\n|\n|\r| )/gm, '')
     const sem = semver.valid(output)
     if (sem) return cb(null, sem)
-    cb(null, new Error(output))
+    cb(new Error(output), null)
```

Comment:
I thought about doing it in a different commit, but at the end I made all in one because the change is related to the install function, I wanted to standardize the callback in every part of the "pipeline" with `(error, version)`.
`browser.js -> install.js -> check-node.js -> browser.js`

I hope I made myself clear :)